osidb/tests/factories.py

```diff
@@ -44,6 +44,11 @@ class Meta:
     description = factory.LazyAttribute(lambda c: f"Description for {c.cve_id}")
     statement = factory.LazyAttribute(lambda c: f"Statement for {c.cve_id}")
     embargoed = factory.Faker("random_element", elements=[False, True])
+    unembargo_dt = factory.Maybe(
+        "embargoed",
+        yes_declaration=None,
+        no_declaration=factory.Faker("date_time", tzinfo=UTC),
```

Comment:
I wonder what date is generated here. I assume a random date. It should however be a date in the past only - if I understand it correctly that it is set only for public flaws. I am not sure it is ensured here. With a future date we would generate an invalid flaw.

Reply:
makes sense, I will use `factory.Faker("past_datetime", tzinfo=UTC)`.